src/protocol/boolean/bitwise_equal.rs

```diff
@@ -107,7 +107,7 @@ where
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
-enum Step {
+pub enum Step {
```

Comment:
lets make it consistent with other step definitions (pub(crate))

Reply:
fixed!